codalab/worker/download_util.py

```diff
@@ -123,6 +124,8 @@ def _compute_target_info(path, depth):
         result['type'] = 'file'
     elif os.path.isdir(path):
         result['type'] = 'directory'
+        # Retrieve the size of the directory and all its contents
+        result['size'] = get_size(path)
```

Comment:
This could be really slow, which is why we didn't put that in...

Reply:
For getting the size for download, could we just use data_size for the top-level bundle and individual files?

Reply:
Yes, we could do that, though that would mean that getting the target info of a directory within a bundle would still return 4096.

Reply:
That's fine or we can make it return `None` or something to not be confusing.